src/output/render/times.rs

```diff
@@ -8,23 +8,39 @@ use crate::output::cell::TextCell;
 use crate::output::time::TimeFormat;
 
 use chrono::prelude::*;
+use chrono_tz::Tz;
 use nu_ansi_term::Style;
 
 pub trait Render {
-    fn render(self, style: Style, time_offset: FixedOffset, time_format: TimeFormat) -> TextCell;
+    fn render(self, style: Style, time_format: TimeFormat) -> TextCell;
 }
 
 impl Render for Option<NaiveDateTime> {
-    fn render(self, style: Style, time_offset: FixedOffset, time_format: TimeFormat) -> TextCell {
-        let datestamp = if let Some(time) = self {
+    fn render(self, style: Style, time_format: TimeFormat) -> TextCell {
+        let datestamp = if let Ok(timezone_str) = iana_time_zone::get_timezone() {
```

Comment:
It may be the case that running `iana_time_zone::get_timezone()` for each render all gives us a constant O(n) penalty here that could be a O(1) if we just put the timezone into memory if it will be displayed. Could be e.g. a static oncelock or something else, assuming eza will never run long enough for a timezone change happening while running being fatal (in fact, it may be preferable so that we always assume the tz from when eza was started vs. having two timezones in the very unlikely crossover case).

Reply:
Yes, that is an issue. I'll see what we can do in this area. The assumption that the TZ will not change within the runtime of `eza` is justified.

Reply:
The code is now checked in, could you repeat the benchmarks?

Reply:
If we knew something about this `dbc` directory, we could run perd tests ourselves. Size, number of files, ....